walletui.py

```diff
@@ -0,0 +1,103 @@
+import ecdsa
+import hashlib
+import requests
+from functools import reduce
+
+
+KEYSTORE = "keys.txt"
+
+
+def gen_key_pairs(n):
+    with open(KEYSTORE, "w") as keyfile:
+        for i in range(n):
+            sk = ecdsa.SigningKey.generate()
+            pk = sk.verifying_key
+            skhex = sk.to_string().hex()
+            pkhex = pk.to_string().hex()
+            keyfile.write(f"{pkhex}\n{skhex}\n\n")
+
+
+def main():
```

Comment:
Why are we defining a main function inside django app?
Why don't we create a management command for it? Or we automate all of it when we hit the default url for the app.

Reply:
this file is essentially some sort of user interface for our blockchain and not a core part of it...think of it like your wallet on some blockchain.... you use it to get info or post info to the blockchain....but it is not "part of the blockchain"

Reply:
You can build a separate interface for it as a separate app on your project. This seems a really inefficient and scriptive way to do things.